diff -aurBb /home/httpd/htdocs/users/rastorguev/manew131/var/ait_patch_orig/compare84612429729784a16433244fa8 /home/httpd/htdocs/users/rastorguev/manew131/var/ait_patch_modif/compare84612429729784a16433244fa8
--- gallery.phtml	2011-02-03 15:33:40 +0200
+++ gallery_.phtml	2011-05-18 09:45:37 +0300
@@ -61,6 +61,7 @@
             <?php foreach ($_block->getImageTypes() as $typeId=>$type): ?>
             <th><?php echo $type['label'] ?></th>
             <?php endforeach; ?>
+            <th><?php echo Mage::helper('aitcg')->__('Custom Product') ?><br />&nbsp;&nbsp;[STORE VIEW]</th>
             <th><?php echo Mage::helper('catalog')->__('Exclude') ?></th>
             <th class="last"><?php echo Mage::helper('catalog')->__('Remove') ?></th>
         </tr>
@@ -73,6 +74,7 @@
                 <?php foreach ($_block->getImageTypes() as $typeId=>$type): ?>
                 <td class="cell-<?php echo $typeId ?> a-center"><input <?php if($_block->getElement()->getAttributeReadonly($typeId)) :?> disabled="disabled" <?php endif;?> type="radio" name="<?php echo $type['field'] ?>" onclick="<?php echo $_block->getJsObjectName(); ?>.setProductImages('__file__')" value="__file__" /></td>
                 <?php endforeach; ?>
+                <td class="cell-cgimage a-center"><input type="checkbox" <?php if($_block->getElement()->getReadonly()):?> disabled="disabled"<?php endif;?> onclick="<?php echo $_block->getJsObjectName(); ?>.updateImage('__file__')" /></td>
                 <td class="cell-disable a-center"><input type="checkbox" <?php if($_block->getElement()->getReadonly()):?> disabled="disabled"<?php endif;?> onclick="<?php echo $_block->getJsObjectName(); ?>.updateImage('__file__')" /></td>
                 <td class="cell-remove a-center last"><input type="checkbox" <?php if($_block->getElement()->getReadonly()):?> disabled="disabled"<?php endif;?> onclick="<?php echo $_block->getJsObjectName(); ?>.updateImage('__file__')" /></td>
         </tr>
@@ -90,6 +92,7 @@
                 </td>
                 <?php endforeach; ?>
                 <td>&nbsp;</td>
+                <td>&nbsp;</td>
                 <td class="last">&nbsp;</td>
         </tr>
     <?php endif ?>
@@ -100,6 +103,7 @@
                 <?php foreach ($_block->getImageTypes() as $typeId=>$type): ?>
                 <td class="cell-<?php echo $typeId ?> a-center"><input type="radio" <?php if($_block->getElement()->getAttributeReadonly($typeId)) :?> disabled="disabled" <?php endif;?> name="<?php echo $type['field'] ?>" onclick="<?php echo $_block->getJsObjectName(); ?>.setProductImages('no_selection')" value="no_selection" /></td>
                 <?php endforeach; ?>
+                <td class="cell-cgimage"><input type="hidden" />&nbsp;</td>
                 <td class="cell-disable"><input type="hidden" />&nbsp;</td>
                 <td class="cell-remove last"><input type="hidden" />&nbsp;</td>
         </tr>
@@ -120,6 +124,10 @@
 <input type="hidden" id="<?php echo $_block->getHtmlId() ?>_save_image" name="<?php echo $_block->getElement()->getName() ?>[values]" value="<?php echo $_block->escapeHtml($_block->getImagesValuesJson()) ?>" />
 <script type="text/javascript">
 //<![CDATA[
+/*
 var <?php echo $_block->getJsObjectName(); ?> = new Product.Gallery('<?php echo $_block->getHtmlId() ?>', <?php if ($_block->getElement()->getReadonly()):?>null<?php else:?><?php echo $_block->getUploader()->getJsObjectName() ?><?php endif;?>, <?php echo $_block->getImageTypesJson() ?>);
+*/
+var removeAitcgConfirmText = "<?php echo Mage::helper('aitcg')->__('By deleting \'Custom Product Preview\' image product option may not work') ?>";
+var <?php echo $_block->getJsObjectName(); ?> = new Product.AitGallery('<?php echo $_block->getHtmlId() ?>', <?php if ($_block->getElement()->getReadonly()):?>null<?php else:?><?php echo $_block->getUploader()->getJsObjectName() ?><?php endif;?>, <?php echo $_block->getImageTypesJson() ?>);
 //]]>
 </script>
